website/docs/for-android/how-to/using-the-portals-plugin.md

```diff
@@ -0,0 +1,345 @@
+
+    // listen on the topic "dismiss" and act on the result data.
+    // This is an example to dismiss a containing native DialogFragment.
+    PortalsPlugin.subscribe("dismiss", (subscriptionResult -> {
+        if(subscriptionResult.getData().equals("cancel")
+            || subscriptionResult.getData().equals("success")) {
+
+            this.dismiss();
+        }
+
+        return Unit.INSTANCE;
+    }));
+
+    // ...
+
+    // add the PortalFragment to the example container fragment
+    final FragmentManager fragmentManager = getChildFragmentManager();
+    fragmentManager.beginTransaction().replace(R.id.example_web_app, portalFragment).commit();
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+**Subscribe Using Annotations**
+
+Android also provides a way to link subscribers defined as methods with annotations.
+
+<Tabs
+defaultValue="kt"
+values={[
+{ label: 'Kotlin', value: 'kt', },
+{ label: 'Java', value: 'java', },
+]}>
+<TabItem value="kt">
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    // ...
+
+    val portal = PortalManager.getPortal("example")
+    val portalFragment = PortalFragment(portal)
+
+    // register this class as having defined message subscribers
+    portalFragment!!.linkMessageReceivers(this)
+
+    // ...
+
+    // add the PortalFragment to the example container fragment
+    val fragmentManager = childFragmentManager
+    fragmentManager.beginTransaction().replace(R.id.example_web_app, portalFragment!!).commit()
+}
+
+/**
+ * Define a `dismiss` method to receive messages on the "dismiss"
+ * topic and act on the message. This is an example to dismiss
+ * a containing native DialogFragment.
+ */
+@PortalMethod
+fun dismiss(result: String?) {
+    if (result != null && (result == "cancel" || result == "success")) {
+        this.dismiss()
+    }
+}
+```
+
+</TabItem>
+
+<TabItem value="java">
+
+```java
+@Override
+public void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // ...
+
+    Portal portal = PortalManager.getPortal("example");
+    PortalFragment portalFragment = new FadePortalFragment(portal);
+
+    // register this class as having defined message subscribers
+    portalFragment.linkMessageReceivers(this);
+
+    // ...
+
+    // add the PortalFragment to the example container fragment
+    final FragmentManager fragmentManager = getChildFragmentManager();
+    fragmentManager.beginTransaction().replace(R.id.example_web_app, portalFragment).commit();
+}
+
+/**
+ * Define a `dismiss` method to receive messages on the "dismiss"
+ * topic and act on the message. This is an example to dismiss
+ * a containing native DialogFragment.
+ */
+@PortalMethod
+public void dismiss(String result) {
+    if(result != null && (result.equals("cancel") || result.equals("success"))) {
+        this.dismiss();
+    }
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+#### React Native
```

Comment:
I think this is covered in the RN section https://github.com/ionic-team/ionic-portals/pull/186/files#diff-a8b064a7489b959fd5a51daabd00b0ba5d40e7f2fad133b8f12e4ac2d5aca1c2